.ci/install_qemu.sh

```diff
@@ -19,6 +19,8 @@ PREFIX=${PREFIX:-/usr}
 KATA_DEV_MODE="${KATA_DEV_MODE:-}"
 
 CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.version")
+echo CURRENT_QEMU_VERSION: $CURRENT_QEMU_VERSION
```

Comment:
Nit: not sure whether you intended to include it in the patch.

Reply:
You caught it before it repushed :) Spurous debug from previous work. Have gotten rid of it now.

Reply:
@likebreath Can you add retest your sysctl test PR by adding a depends-on on this PR?

Reply:
@amshinde I believe we can't do depends on for the same repo (sysctl test PR #2901). So we may have to land this one first.